angular/projects/admin-nrpti/src/app/mines/mines-records-detail/mines-records-detail.component.html

```diff
@@ -42,6 +42,8 @@ <h2 class="border-0 mb-0">Record Information</h2>
                 ? 'Original Permit'
                 : record && record.typeCode && record.typeCode === 'AMD'
                 ? 'Permit Amendment'
+                : record && record.typeCode && record.typeCode === 'ALG'
+                ? 'Amalgamated Permit'
```

Comment:
This ternary could also be addressed if you do decide to pull out a function and go the route of early returns.